src/features/gateway/steps/flows/ReleaseStandardStatuses.tsx

```diff
@@ -203,6 +197,22 @@ export const ReleaseStandardBurnProgressStatus: FunctionComponent<
             />
           </>
         )}
+        {renVMStatus === ChainTransactionStatus.Reverted && (
+          <Box mt={3} mb={3}>
+            <Typography variant="h3" align="center">
+              RenVM {t("tx.reverted")}
+            </Typography>
+          </Box>
```

Comment:
Create a reusable component for this status so that we can share it across multiple flows, while controlling its behaviour in one place.